app/javascript/bundles/User/components/UserShow.jsx

```diff
@@ -19,7 +19,7 @@ function UserShowView ({ user, subscription, streak }) {
     <>
       {subscription && subscription.active && streak && (
         <p className='notice--subscription'>
-          You have been subscribed for {streak}
+          You have been a member for {streak} month{streak > 1 ? 's' : ''}
```

Comment:
This can be done using this `pluralize` https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize. Also, we should be using i18n for these strings.

Reply:
This is inside a JSX component, so we don't have access to these rails methods.

Reply:
Oh ok cool, let’s leave it like this then